src/RoomsSDKAdapter.js

```diff
@@ -304,11 +304,11 @@ export default class RoomsSDKAdapter extends RoomsAdapter {
           logger.debug('ROOM', ID, 'getActivitiesInRealTime()', ['received "event:conversation.activity" event', {sdkActivity}]);
 
           const activity = {
```

Comment:
Need to verify if this method should invoke `fromSDKActivity` in the `ActivitiesAdapter` instead of inlining.